src/transaction/creation_common.rs

```diff
@@ -522,21 +524,18 @@ pub(super) mod transaction_path {
     const ARG_VALUE_NAME: &str = common::ARG_PATH;
     const ARG_HELP: &str = "Path to input transaction file";
 
-    pub fn arg() -> Arg {
+    pub fn arg(required: bool) -> Arg {
```

Comment:
I see this called as `arg(true)`, but I don't see `arg(false)`. Is there a point in having this as a parameter if it's always true?